guides/source/configuring.md

```diff
@@ -140,6 +140,8 @@ defaults to `:debug` for all environments. The available log levels are: `:debug
 
 * `secrets.secret_key_base` is used for specifying a key which allows sessions for the application to be verified against a known secure key to prevent tampering. Applications get `secrets.secret_key_base` initialized to a random key present in `config/secrets.yml`.
 
+* `config.secrets_cipher` allows you to change the cipher used to protect the `config/secrets.yml` file. This defaults to `"aes-128-gcm"`. Versions of OpenSSL earlier than 1.0.01, unfortunately, don't support GCM mode, so degrading this to `"aes-128-cbc"` may be necessary. If you find yourself in this situation, strongly consider updating OpenSSL to a modern version if at all possible rather than weakening this setting.
```

Comment:
@wlipa if you encrypt everything, then everyone needs the decryption keys. Maybe I'm misunderstanding people's desired use case here. To me, it seems better to keep "not actually so secret" values, like keys for mock/test APIs, in cleartext where everyone who needs them can see them, while "actually secret" values are encrypted by a key that's only known by the production environment.